cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -36,45 +36,60 @@ if("@MSVC@")
 endif()
 
 # Same logic in all url_sha1_boost* schemes (TODO: DRY)
```

Comment:
> TODO: DRY

Can you check that something like this fits `hunter_get_boost_libs(VERSION @HUNTER_Boost_VERSION@ LIBS boost_libs)`? In this case logic can be moved to `hunter_get_boost_libs.cmake` and all schemes will use only this function.